.github/workflows/build_bundle_package.yml

```diff
@@ -35,12 +35,15 @@ on:
         required: true
         default: '3.3.3'
 
+permissions:  # added using https://github.com/step-security/secure-repo
+  contents: read
+
 jobs:
   build-native-lib:
     runs-on: ubuntu-20.04
     container: inteldpo/gluten-centos-packaging:latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

Comment:
v2 is picked as it's not able to support high nodejs version by default on our cent7 based image